R/profilePlot.R

```diff
@@ -246,36 +195,13 @@ profilePlot <- function(contrastDF,
         referenceLine <- "darkgoldenrod1"
     }
 
-    if (any(is.null(refLineThickness),
-            !is.numeric(refLineThickness),
-            length(refLineThickness) != 1,
-            refLineThickness < 0)) {
-        warning("refLineThickness must be a singular value of class numeric Assigning default value '1'.")
-        refLineThickness <- 1
-    }
-
-    if (!is.null(legendPosition) &&
-        !all(is.character(legendPosition),
-             length(legendPosition) == 1,
-             legendPosition %in% c("top", "bottom", "left", "right", "ne", "se", "nw", "sw"))) {
-        warning("legendPosition must be one value from 'top', 'bottom', 'left', 'right', 'ne', 'se', 'nw', 'sw' or 'NULL' to disable. Assigning default value 'right'.")
-        legendPosition <- "right"
-    }
-
-    if (missing(footnote)) {
-        footnote <- NULL
-    } else if (!is.null(footnote) &&
-               !all(is.character(footnote), length(footnote) == 1)) {
-        warning("footnote must be a singular value of class character or 'NULL' to disable. Assigning default value 'NULL'.")
-        footnote <- NULL
-    }
-
     if (any(is.null(sizeBySignificance),
             !is.logical(sizeBySignificance),
             length(sizeBySignificance) != 1)) {
         warning("sizeBySignificance must be a singular logical value. Assigning default value FALSE")
         sizeBySignificance = FALSE
     }
+
     # Columns to plot
     # Capture the labels from the colname
     xlabel <- make.names(logIntCol)
```

Comment:
why not using make names on all column names one time?

Reply:
this is did by you only so I cant get your point mohammed.